public/app/features/alerting/state/alertDef.ts

```diff
@@ -158,9 +151,16 @@ function getStateDisplayModel(state: string) {
         stateClass: 'alert-state-critical',
       };
     }
-  }
 
-  throw { message: 'Unknown alert state' };
+    default:
```

Comment:
Is there a general style preference for ordering of fallthrough statements? I'm fine with whatever, but this would look odd to me in Scala, for example.

Reply:
By convention it's the last one but doesn't need to be, I'll swap these two lines so we keep the code consistent everywhere 👍